include/CppUTest/StandardCLibrary.h

```diff
@@ -15,6 +15,8 @@
 #ifdef __cplusplus
  #if CPPUTEST_USE_STD_CPP_LIB
   #include <cstdlib>
+  #include <vector>
+  #include <algorithm>
```

Comment:
We don't really want to depend that much on the STD C++ library. Is there a way around that?

Reply:
Fixed using defines

Reply:
Fixed